modules/templates/helper.go

```diff
@@ -135,6 +135,7 @@ func NewFuncMap() []template.FuncMap {
 		"CountFmt":      base.FormatNumberSI,
 		"TimeSince":     timeutil.TimeSince,
 		"TimeSinceUnix": timeutil.TimeSinceUnix,
+		"DateTime":      timeutil.DateTime,
```

Comment:
What do you think about #24266 ? Then we can have a `TimeUtils.DateDate`, and move all date/time related helper functions into `TimeUtils`, to simplify the helper.go.

Reply:
Whichever PR lands first will adopt the other

Reply:
Looks like #24266 is already in the merge queue. I think that moving all date/time related should happen in a PR of its own

Reply:
https://github.com/go-gitea/gitea/pull/24266 is inö

Reply:
Yes but I don't want to bloat this PR

Reply:
ok